src/registers/id_aa64dfr0_el1.rs

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Copyright (c) 2024 by the author(s)
+//
+// Author(s):
+//   - Sangwan Kwon <sangwan.kwon@samsung.com>
+
+//! AArch64 Debug Feature Register 0 - EL1
+//!
+//! Provides top level information about the debug system in AArch64 state.
+
+use tock_registers::{interfaces::Readable, register_bitfields};
+
+register_bitfields! {u64,
+    pub ID_AA64DFR0_EL1 [
+        /// Branch Record Buffer Extension.
+        BRBE OFFSET(52) NUMBITS(4) [],
+
+        /// Multi-threaded PMU extension.
+        MTPMU OFFSET(48) NUMBITS(4) [],
+
+        /// Trace Buffer Extension.
+        TraceBuffer OFFSET(44) NUMBITS(4) [],
```

Comment:
This can be converted into an enum, only 3 valid values for 4 available bits.

Reply:
I reflected it !